README.md

```diff
@@ -487,11 +493,43 @@ type T = {
   readonly a: number;
   b: string;
 };
-type Result = WritableKeys<T>
+type Result = WritableKeys<T>;
 // Result:
 // "b"
 ```
 
+### OptionalKeys
+
+Gets keys of an object which are optional.
+
+```typescript
+type T = {
+  a: number;
+  b?: string;
+  c: string;
```

Comment:
we could make one key "undefinable", so it's clearer for the unsure readers that this type works on the keys not on value type
```suggestion
  c: string | undefined;
```